config/matrix-api.yml

```diff
@@ -125,4 +141,21 @@ paths:
             type: array
             items:
               $ref: '#/definitions/MatrixFormat'
-
+  /dss/notifications:
+    post:
+      summary: "Handles DSS notifications such as creates, updates and deletes."
+      description: "Handles DSS notifications such as bundle creates, updates and deletes and propagates changes to
```

Comment:
Seems odd to make a reference to redshift (implementation detail) in the API spec?

Reply:
Ah, yes revising